tests/test_schemas.py

```diff
@@ -83,19 +84,19 @@ def getInvalidValue(self, cls, fieldName):
         Returns a value that should trigger a schema validation failure.
         """
         field = self.getAvroSchema(cls, fieldName)
-        t = field.type
-        if isinstance(t, avro.schema.UnionSchema):
-            t = t.schemas[1]
-        if isinstance(t, avro.schema.MapSchema):
+        typ = field.type
```

Comment:
Actually, reading pep8 I see that the recommended way of doing this is to use a trailing underscore:
```
single_trailing_underscore_ : used by convention to avoid conflicts with Python keyword, 
```
By this, we should have `class_` instead of `cls` and `type_` instead of `typ`. What's your preference? I don't have a strong opinion about it, I just thought I should bring it up.